mava/components/jax/building/loggers.py

```diff
@@ -40,11 +41,13 @@ def __init__(
     def on_building_executor_logger(self, builder: SystemBuilder) -> None:
         """[summary]"""
         logger_config = self.config.logger_config if self.config.logger_config else {}
+        # is_evaluator set by builder
```

Comment:
I just realised that all the comments could have been put in on purpose and aren't left over from you reading through the code. If that is the case, please ignore my other comments.